src/context/apolloCache.ts

```diff
@@ -0,0 +1,23 @@
+import { InMemoryCache } from '@apollo/client';
+
+import { colonyCache } from '~cache';
+
+/*
+ * @NOTE Make any modifications to GraphQL query data in here (replaces old client-only resolvers)
+ *
+ * See: https://www.apollographql.com/docs/react/local-state/managing-state-with-field-policies
+ */
+const localOnlyFields = {
```

Comment:
I think calling this `localOnlyFields` is a bit misleading. What we seem to be doing is modifying the fields returned from the API, rather than adding fields that are only present locally (which is also possible, and may be the case in the future)

Reply:
Sure. Will rename